Sources/Build/PackageVersion.swift

```diff
@@ -15,7 +15,7 @@ import class Utility.Git
 import struct Utility.Path
 
 public func generateVersionData(_ rootDir: AbsolutePath, rootPackage: Package, externalPackages: [Package]) throws {
-    let dirPath = rootDir.appending(".build/versionData")
+    let dirPath = rootDir.appending(components: ".build", "versionData")
```

Comment:
This is a change I like.  I actually had on my list to go through and unwind all of these cases of embedded `/` characters, so this change I think is a good one.